lib/vrl/stdlib/Cargo.toml

```diff
@@ -34,6 +34,8 @@ uuid = { version = "0.8", features = ["v4"], optional = true }
 
 [dev-dependencies]
 anyhow = "1"
+criterion = "0.3"
+shared = { path = "../../shared", features = ["btreemap"] }
```

Comment:
I'd rather not depend on internal non-VRL crates in here, as it'll make splitting off VRL into its own repo more of a pain in the future.

In VRL we have the `map` macro available for this (we chose that name, since VRL itself has no concept of `Value::Map` (it uses the name `Object`), so there's no naming conflict here.

The macro is exported via the prelude, so should be available already in the bench file.

Reply:
Actually, it looks like this is only used in `lib/remap-functions/src/parse_url.rs`, but we moved that file, so once that is resolved, this dependency can be removed in its entirety, and there's no need to use `map!` anywhere else.

Reply:
Ah, never mind me, we already depend on `shared`, so we'll have to do some work to get that out of here either way.

Still though, it shouldn't be needed as a dev-dependency as far as I can tell.

Reply:
Yeah, I think this was another bad merge. Thanks for catching!